Pod/Classes/MWPhotoBrowser.m

```diff
@@ -372,6 +372,9 @@ - (void)viewWillAppear:(BOOL)animated {
     }
     
     // Layout
+    if (@available(iOS 11.0, *)) {
+        [self layoutVisiblePages];
+    }
     [self.view setNeedsLayout];
```

Comment:
Is there an explanation for why `layoutVisiblePages` needs to be called before`viewWillLayoutSubviews` is called on iOS 11? either way, imo we should add an explanation or reference to this bug fix.

Reply:
there is no need thats called before or after 
it should called only once 
It looks like that in iOS 11 the removeFromSuperview function will call every time viewWillLayoutSubviews.